libc/unistd_getopt.c

```diff
@@ -314,6 +314,33 @@ TEST(unistd_getopt, getopt_unknownopt_noarg)
 }
 
 
+TEST(unistd_getopt, getopt_unknownopt_multiple)
+{
+	/* mocks of main() arguments */
+	char *testargv[] = { "cmd", "-xxx", "", "" };
+	testargc = 2;
+
+	testmain(testargc, testargv, &ret, "abc:");
+	TEST_ASSERT_EQUAL_INT(UNKNOWNOPT, ret.err);
+	TEST_ASSERT_EQUAL_INT(0, ret.aflag);
+	TEST_ASSERT_EQUAL_INT(0, ret.bflag);
+	TEST_ASSERT_NULL(ret.cvalue);
+	TEST_ASSERT_EQUAL_INT(0, ret.nonopts);
+
+	/* Assert that parsing argument work correctly after passing multiple unknown options */
+	testargv[1] = "-ab";
+	testargv[2] = "-c";
+	testargv[3] = "value";
+	testargc = 4;
+
+	testmain(testargc, testargv, &ret, "abc:");
```

Comment:
invalid - calling `getopt` again without resetting it, also without resetting `ret` struct.

Also: make `ret` struct static